rust/sdk-processor/src/steps/common/parquet_version_tracker_step.rs

```diff
@@ -0,0 +1,161 @@
+use crate::parquet_processors::ParquetTypeEnum;
+use anyhow::Result;
+use aptos_indexer_processor_sdk::{
+    traits::{
+        pollable_async_step::PollableAsyncRunType, NamedStep, PollableAsyncStep, Processable,
+    },
+    types::transaction_context::{TransactionContext, TransactionMetadata},
+    utils::errors::ProcessorError,
+};
+use async_trait::async_trait;
+use std::collections::HashMap;
+use tracing::debug;
+
+/// The `ParquetProcessorStatusSaver` trait object should be implemented in order to save the latest successfully
+///
+/// processed transaction versino to storage. I.e., persisting the `processor_status` to storage.
```

Comment:
typo
```suggestion
/// processed transaction version to storage. I.e., persisting the `processor_status` to storage.
```

Reply:
is the empty line on 15 intended?

Reply:
lint was warnning it, reworded and fixed!